dataAccessObjects.py

```diff
@@ -61,6 +61,13 @@ def checkUserExisted(self,userName):
             + 'WHERE user_name="{userName}"'.format(userName=userName)
         return self.queryOneValue(sql)
 
+    def createNewUser(self,userName,userPassword):
+        cursor = self.db.cursor
+        sql = 'INSERT INTO user (user_name,user_password) VALUES (%s,%s)'
+        data = (userName,userPassword)
+        ret = cursor.execute(sql,data)
```

Comment:
I have tested it, only this method is available.
Below is the error message when I using cmd function.
<img width="884" alt="2017-07-26 10 34 08" src="https://user-images.githubusercontent.com/15280812/28626907-dd47663a-7252-11e7-908c-c99f90b64a10.png">